<launch>
  <node type="revolution_command" pkg="surf_rec" name="revolution_command" output="screen" />

  <node type="point_cloud2_assembler" pkg="laser_assembler" name="kinect_point_cloud_assembler" output="screen">
    <!-- <remap from="cloud" to="point_cloud_kinect_link_frame" /> -->
    <remap from="cloud" to="/camera/depth/points" />
    <param name="tf_cache_time_secs" type="double" value="100.0" />
    <param name="max_clouds" type="int" value="1000" />
    <param name="fixed_frame" type="string" value="base_link" />
  </node>
</launch>
